internal/app/api/api.go

```diff
@@ -72,6 +72,23 @@ func Init(conns *InfraConns) (http.Handler, error) {
 			method:  post,
 			handler: userHandler.Login,
 		},
+		route{
+			path:        "/getme",
```

Comment:
Follow this page to update the path: https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/